src/test/java/com/training/services/tests/GAAccountTest.java

```diff
@@ -65,20 +68,23 @@ public void testAuthenticateWrongAppKey() {
                 isEqualTo("The API key header is required and the API key provided should be valid.");
     }
 
+    /**
+     * testAuthenticateWrongUsername():post requested with wrong Username and checked the negative response,
+     * parameters values passed through Json
+     */
     @Test
     public void testAuthenticateWrongUsername() {
-        RestAssured.baseURI = map.get("URI");
-        Response authenticationResponse = given().
-                header(map.get("AppKeyHeader"), (map.get("AppKeyValue"))).
-                header(map.get("ContentTypeHeader"), (map.get("ContentTypeValue")))
-                .body("{\n" +
-                        " \"uid\": \"testShri@api.com\",\n" +
-                        " \"password\": \"Password1\"\n" +
-                        "}").post("/authentication/login")
-                .then()
-                .extract()
-                .response()
-                .as(Response.class);
+        Map<String, Object> headerMap = new HashMap();
+        headerMap.put(map.get("AppKeyHeader"), map.get("AppKeyValue"));
+        headerMap.put(map.get("ContentTypeHeader"), map.get("ContentTypeValue"));
```

Comment:
lets use this in before Each and, directly pass the variable headerMap to RestEngine.